tests/assembly/x86_64-function-return.rs

```diff
@@ -0,0 +1,28 @@
+// Test that the function return is (not) converted into a jump to the thunk
+// when the `-Zfunction-return={keep,thunk-extern}` flag is (not) set.
+
+// revisions: unset keep thunk-extern keep-thunk-extern thunk-extern-keep
+// assembly-output: emit-asm
+// compile-flags: -O
+// [keep] compile-flags: -Zfunction-return=keep
+// [thunk-extern] compile-flags: -Zfunction-return=thunk-extern
+// [keep-thunk-extern] compile-flags: -Zfunction-return=keep -Zfunction-return=thunk-extern
+// [thunk-extern-keep] compile-flags: -Zfunction-return=thunk-extern -Zfunction-return=keep
+// only-x86_64
+
+#![crate_type = "lib"]
+
+// CHECK-LABEL: foo:
+#[no_mangle]
+pub unsafe fn foo() {
+    // unset: ret
```

Comment:
In order to mitigate LVI vulnerabilities, the `x86_64-fortanix-unknown-sgx` target must not use `ret` instructons. Every `ret` is rewritten into something like `popq %rax; lfence; jmp *%rax`. Therefore these tests fail on that platform. Can this platform be ignored for these tests please?

Reply:
Added `// ignore-sgx Tests incompatible with LVI mitigations` like you did in other tests.